lib/program-helper.js

```diff
@@ -20,11 +20,13 @@ function error() {
 
 /**
  * Resolve file globs based in command line or tsconfig.json file.
- * @returns {Array}
+ * May throw an error.
```

Comment:
should use JSDoc `@throws` comment here, and check for other places where this may be needed